pset-functions.js

```diff
@@ -175,8 +235,21 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
-// Add your code above this line
+function palindrome(str) {
 
+    var len = str.length;
```

Comment:
@deedeemedina while `var1 works, we generally want to use `const` and `let` to declare our variables since